### PR TITLE
Issue #6726: aligned javadoc/xdoc for SuppressionFilter

### PR DESCRIPTION
**Pull request by pbludov**
Issue #6726 (part of #5750)

No changes in the code.

### CODE REVIEW

**Comment by rnveach** [comment]
> The test XdocsJavaDocsTest should be reviewed to ensure that it is compatible with filters.

Isn't that what you did when you enabled it in this PR?

File: src/xdocs/config_filters.xml [hunk 1]
```diff
@@ -477,8 +481,8 @@ public class UserService {
              </tr>
           </table>
           <p>
```
Comment (pbludov) [inline src/xdocs/config_filters.xml]: This part is missing in the file `SuppressionFilter.java`.
I think the `properties` section should contain only the properties table

Comment (rnveach) [reply src/xdocs/config_filters.xml]: I agree, I couldn't find it but there is an issue somewhere about creating a new section in the check/filter for external configuration files like this.

**Comment by romani** [comment]
image

File: src/xdocs/config_filters.xml [hunk 2]
```diff
@@ -420,17 +420,34 @@ public class UserService {
...
             audit events for Check errors according to
-            a <a href="dtds/suppressions_1_2.dtd"><em>suppressions XML
-            document</em></a> in a file. If there is no configured
+            a <a href="https://checkstyle.org/dtds/suppressions_1_2.dtd">suppressions XML
+            document</a> in a file. If there is no configured
             suppressions file or the optional is set to true and
             suppressions file was not found the Filter accepts all audit events.
           </p>
       </subsection>
+      <subsection name="Notes" id="SuppressionFilter_Notes">
```
Comment (romani) [inline src/xdocs/config_filters.xml]: please move Notes below Properties section.
As we need to show all list of properties first and only below explain nuances. Please make link from `file` at Properties to Notes, something like "See details at Notes" or ...

File: src/xdocs/config_filters.xml [hunk 3]
```diff
@@ -420,17 +420,34 @@ public class UserService {
...
+        <p>
+          The suppression file location is checked in following order:
```
Comment (romani) [inline src/xdocs/config_filters.xml]: Location of file defined in `file` property ....

**Comment by romani** [comment]
@pbludov , ping, please update code according to my notes or support your code.

**Comment by pbludov** [comment]
> @pbludov , ping, please update code according to my notes or support your code.

This PR is blocked by #6809
And there may be one more PR to organize  Xml properties into a table.

**Comment by romani** [comment]
@pbludov , blocker issue is merged/fixed, please rebase.

**Comment by pbludov** [comment]
> @pbludov , blocker issue is merged/fixed, please rebase.

Done. Rebased and fall in an another trap: #6817

File: src/xdocs/config_filters.xml (hunk 3 above)
Comment (pbludov) [reply src/xdocs/config_filters.xml]: done

File: src/xdocs/config_filters.xml (hunk 2 above)
Comment (pbludov) [reply src/xdocs/config_filters.xml]: done

**Comment by pbludov** [comment]
supp

File: src/xdocs/config_filters.xml (hunk 2 above)
Comment (romani) [reply src/xdocs/config_filters.xml]: > Please make link from file at Properties to Notes, something like "See details at Notes" or ...

not done, but could be done any time after this PR.

**Pull request merged**